gocourse03/drugs/main.go

```diff
@@ -0,0 +1,56 @@
+/*
+Треба відсортувати ящик ліків за датою виробництва.
+Застарілі
+Нові
+яким півроку
+*/
+package main
+
+import (
+	"fmt"
+	"github.com/brianvoe/gofakeit/v7"
+	"time"
+)
+
+type Drugs struct {
+	Name         string
+	Manufactured time.Time
+}
+
+func main() {
+	fake := gofakeit.New(0)
+	dateRangeAt := time.Now().AddDate(-2, 0, 0)
+	dateRangeTo := time.Now().AddDate(0, 0, 1)
```

Comment:
Я би ось так назвав змінні:
```suggestion
	fakeStart := time.Now().AddDate(-2, 0, 0)
	fakeEnd := time.Now().AddDate(0, 0, 1)
```


https://pkg.go.dev/github.com/brianvoe/gofakeit#DateRange
```
func DateRange(start, end time.Time) time.Time
```